src/box/vy_scheduler.c

```diff
@@ -356,7 +356,13 @@ vy_worker_pool_stop(struct vy_worker_pool *pool)
 	assert(pool->workers != NULL);
 	for (int i = 0; i < pool->size; i++) {
 		struct vy_worker *worker = &pool->workers[i];
-		cord_cancel_and_join(&worker->cord);
+		cbus_stop_loop(&worker->worker_pipe);
+		cpipe_destroy(&worker->worker_pipe);
+	}
+	for (int i = 0; i < pool->size; i++) {
+		struct vy_worker *worker = &pool->workers[i];
+		if (cord_join(&worker->cord) != 0)
+			panic_syserror("iproto cord join failed");
```

Comment:
You're effectively reverting e463128e4bd40 here. Looks like no using `cbus_stop_loop` was a deliberate decision.

Reply:
Discussed f2f and decided to drop this change from the PR. The reason this and similar change got in the PR is to fix ASAN issues in other place. We decided to temporarily add ASAN suppression for that place.